src/Donations/Components/PaymentsForm.tsx

```diff
@@ -152,7 +152,7 @@ function PaymentsForm({}: Props): ReactElement {
     });
     if (router.query.to) {
       router.replace({
-        query: { to: projectDetails.id, step: PAYMENT },
+        query: { to: projectDetails.slug, step: PAYMENT },
```

Comment:
I think this error got introduced with https://github.com/Plant-for-the-Planet-org/planet-donations/commit/537774eafc043e5d156297bc7bc3e8c7c2852514 when we started replaced the `router.query.to` parameter with the project id.

Unfortunately now using the project slug you will get duplicate donations if you started the donation process not with the slug but the project id, e.g. changing the URL from https://donate-with-planet-bktl5zw35-planetapp.vercel.app/?to=proj_mF3CfwT74kvt4AdRji8X8u0D to https://donate-with-planet-bktl5zw35-planetapp.vercel.app/?to=endowment-plant-for-the-planet-germany. I think we have to keep the parameter constant in order to avoid creating two donations. 

So (1) either whatever we started with in the beginning has to be used also later on or (2) the project id has to be swapped against the slug much earlier in the donation process before doing any backend API calls to create a draft donation. Of course they might be other ways to solve this.

Reply:
We are not using project id anywhere in the app, except payment options and create donation API calls. 
If the user changes query param `to` from project slug to project id, the page is going to refresh which in turn anyway going to create a new donation.
Also, the backend responds with the same response, no matter we pass project slug or project id.

Reply:
E.g. Klumforest.com is using a project id directly with https://donate.plant-for-the-planet.org/?tenant=ten_KRHYP8TR&to=proj_WZkyugryh35sMmZMmXCwq7YY.

Reply:
I hope the commit below will solve this issue